spec/shared/hutch_notifications.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+shared_context "hutch notifications", shared_context: :metadata do
```

Comment:
What's the `shared_context: :metadata` thing? Is it necessary?

Reply:
no, I copy-pasted it :see_no_evil: